### DISCUSSION TITLE
In what household item are you most likely to find the radioactive isotope Americium-241?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#483] [Galaxy Trivia] [answered]
**Hint:** 🚬🕵️

**Comment by ParanoidUser** [comment]
@trevorwang, @newzpanuwat, @ZH1995, @Taky0nDon, anyone? 😶‍🌫️

**Comment by trevorwang** [comment, marked as answer]
Smoke detector?

**Reply by ParanoidUser** [reply]
Bingo! 👏👏 Congratulations on your well deserved [Galaxy Brain](https://github.com/trevorwang?tab=achievements&achievement=galaxy-brain) badge. Continue to unlock the next badge levels (Bronze for 8 answers), and don't hesitate to post your own questions to the community. 😊🎉
